apbs/chemistry/atom_complex_calc.py

```diff
@@ -112,9 +110,7 @@ def atom_surface(self, atom: Atom, ref: Surface, prad: float) -> Surface:
             pos.z = rad(ref.zs[i]) + apos.z
 
             # need to implement
-            if self.accessable_outside_inflated_venderwalls_rad(
-                pos, prad, atomID
-            ):
+            if self.accessable_outside_inflated_venderwalls_rad(pos, prad, atomID):
```

Comment:
This is spelled *very* incorrectly.  I'd recommend `accessible_outside_inflated_vdw_radius` instead.  "venderwalls" should be "vanderwaals" in case my suggestion doesn't stick.

Reply:
Fixex